installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -111,8 +122,55 @@ include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+3]
 * xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[Changing the MTU for the cluster network]
 * xref:../../networking/changing-cluster-network-mtu.adoc#nw-ovn-ipsec-enable_configuring-ipsec-ovn[Enabling IPsec encryption]
 
+// Opting in to AWS Local Zones
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+1]
+
+.Next steps
+
+Choose one of the following options to install an {product-title} cluster in an AWS Local Zone environment:
```

Comment:
Added a next steps section as this is the junction whereby a customer can choose the Full IPI path of the existing VPC path.

Reply:
Updated, but I had to changed `Installing a cluster in an existing VPC with Local Zone subnets` to `Installing a cluster in an existing VPC with defined Local Zone subnets` because it would read like one can install a cluster in an existing VPC by using LZ subnets; that is, the LZ subnets to the installing activity.

Reply:
Okay, sorry to press the key here, but it is the section that will help users to decide what they want to do.

I didn't follow correctly the `defined` here.

Let's review the existing section for creating the VPC "[About using a custom VPC"](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc_installing-aws-vpc), it says:

> you can deploy a cluster into existing subnets in an existing Amazon Virtual Private Cloud (VPC) in Amazon Web Services (AWS). 

It describes all the dependencies: `subnets -- in -> VPC -- in --> AWS`

The title of that page is "Installing a cluster on AWS into an existing VPC". In practice, both are connected with the goal. When bringing to the Local Zones, it seems interesting to connect with existing docs and adapt to Local Zones. 

Local Zones are the service Name but also locations, data centers outside the regular region closer to the metropolitan regions, extending the cloud to the big cities.

With that in mind, it seems interesting to use `existing subnets` and/or `extending` terms, something like:

- Installing a cluster in existing subnets extending to Local Zones

or just this - considering Local Zone is a location external to regular subnets/zones:

- Installing a cluster in existing subnets in Local Zones

Is it make sense?

Reply:
Thanks, @mtulio , for the explanation. Makes sense, but if I changed `Installing a cluster in an existing VPC with defined Local Zone subnets` to one of your suggested titles, the "existing VPC" part will be missing from the title. This part of the title is vital to differentiate it as an alternative route to `Installing a cluster quickly in AWS Local Zones`?

Reply:
@dfitzmau you can keep the existing VPC, but as I commented above, I didn't understand what do you mean by using `defined`?

Reply:
@mtulio . The placement of "with" suggests that one can install a cluster in an existing by using Local Zone subnets. The LZ subnets are what makes installing a cluster in an existing VPC possible. This is why I mentioned "defined" to break the word placement 

Example of the placement of "with" in published OCP docs: https://docs.openshift.com/container-platform/4.13/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html

Maybe: `Installing a cluster in an existing VPC that contains Local Zone subnets`

Reply:
ACK